docs/diagrams/ModelClassDiagram.puml

```diff
@@ -36,16 +45,25 @@ AddressBook .up.|> ReadOnlyAddressBook
 
 ModelManager .up.|> Model
 Model .right.> ObservableList
+Model .right.> ObservableMap
+ObservableList -[hidden]down- ObservableMap
 ModelManager o--> "1" AddressBook
 ModelManager o-left-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
 AddressBook *--> "1" UniquePersonList
+AddressBook *.left.--> "1" GroupHashMap
+
+GroupHashMap o--> "*" Group
+
 UniquePersonList o--> "*" Person
 Person *--> Name
 Person *--> Phone
 Person *--> Email
 Person *--> Address
+Person *--> Birthday
+Person --> Event
```

Comment:
Could we add a "*" multiplicity to `Event`? Might be good to use labels here as well since there are 2 links to Event: `dates` and `meetings`